packages/core/src/layout/ItemCard/ItemCardHeader.tsx

```diff
@@ -77,8 +77,16 @@ export const ItemCardHeader = (props: ItemCardHeaderProps) => {
   const classes = useStyles(props);
   return (
     <div className={classes.root}>
-      {subtitle && <Typography variant="subtitle2">{subtitle}</Typography>}
-      {title && <Typography variant="h6">{title}</Typography>}
+      {subtitle && (
+        <Typography variant="subtitle2" component="h3">
```

Comment:
So, these are in a surprising order, where the subtitle is above the title. I just want to double check, should it really be the case that this shall be an h3 then, although it's "smaller" in scope than the title? Or should this be  h4 and the title h3

Reply:
Hi @freben  We did that change because We don't want to break the current ui. if we  modify as  you said we are going to have this result.. What do you think???   Thanks!!!
<img width="1719" alt="Screen Shot 2021-04-05 at 8 51 09" src="https://user-images.githubusercontent.com/24494837/115084332-5b786300-9ece-11eb-9ae6-86f7d9a91b58.png">

Reply:
OK all good from my side then - if you get the desired screen reader end result out of this and it looks correct, I don't mind!